app/src/main/java/com/github/multimatum_team/multimatum/ReminderBroadcastReceiver.kt

```diff
@@ -6,29 +6,35 @@ import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
 import android.content.SharedPreferences
-import android.util.Log
-import androidx.preference.PreferenceManager
+import androidx.annotation.CallSuper
 import androidx.core.app.NotificationCompat
 import com.github.multimatum_team.multimatum.activity.MainActivity
 import com.github.multimatum_team.multimatum.repository.DeadlineRepository
+import dagger.hilt.android.AndroidEntryPoint
 import javax.inject.Inject
 
+
 /**
  * This class is a BroadcastReceiver that will send a notification when it is triggered
  */
+@AndroidEntryPoint
 class ReminderBroadcastReceiver : BroadcastReceiver() {
+
     @Inject
-    lateinit var deadlineRepository: DeadlineRepository
+    lateinit var preferences: SharedPreferences
```

Comment:
```suggestion
    @Inject
    lateinit var preferences: SharedPreferences
```

Reply:
there's the `@Inject` from line 23 that stands there already, since I removed a line. It ends up as expected.

Reply:
Oh indeed, sorry!